packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -552,7 +552,9 @@ export const Dropdown = (React.forwardRef<HTMLDivElement, DropdownProps>((props,
       },
       'aria-invalid': ariaInvalid,
       'aria-label': undefined,
-      'aria-labelledby': [ariaLabelledby, triggerButtonId].filter(l => !!l).join(' '),
+      ...(ariaLabelledby && {
+        'aria-labelledby': [ariaLabelledby].filter(l => !!l).join(' '),
```

Comment:
since you're removing the `triggerButtonId` there's no more reason to keep the array operations

```suggestion
      ...(ariaLabelledby && {
        'aria-labelledby': ariaLanelledby,
```

Reply:
updated